src/core/services/LabService.ts

```diff
@@ -129,7 +128,7 @@ export class LabServiceImpl implements LabService {
               }
             : undefined,
           displayName: backscratcherData.name,
-          displayIcon: `${ASSET_URL}${YVECRV}/logo-128.png`,
+          displayIcon: `${getConstants().ASSETS_ICON_URL}${YVECRV}/logo-128.png`,
```

Comment:
nit: how about destructuring the object to access the property needed from `getConstants()`, since its used multiple times?